docs/labs/12-Hash.md

```diff
@@ -118,16 +118,43 @@ Once in the HashMap,
 you can use ```get``` to retrieve the definition in the future.
 
 ```java
-String def = phobias.get("ergophobia")
+String def = phobias.get("ergophobia");
 // def now has the value "the abnormal fear of work"
 ```
 
+In the case of say, a flashcard application,
+the definition can be retrieved to return the key using a ```getKey``` function.
+This method does require a for each loop to check all hashmap entries,
+as well as the importing of the Map.entry java utility.
+The import allows us to use the ```Entry``` object
+as well as the ```map.entrySet``` method in order to iterate each hashmap entry.
+(Resources listed below code segment)
```

Comment:
There is a way to do a footnote See here <https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/>
You can then delete the line 131 and then place the footnotes for the code and what you mention there down at the bottom.